release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3127,6 +3127,8 @@ In the following tables, features are marked with the following statuses:
 
 * When using CPU load balancing on cgroupv2, a pod can fail to start if another pod that has access to exclusive CPUs already exists. This can happen when a pod is deleted and another one is quickly created to replace it. As a workaround, ensure that the old pod is fully terminated before attempting to create the new one. (link:https://issues.redhat.com/browse/OCPBUGS-34812[*OCPBUGS-34812*]) 
 
+* Enabling LUKS encryption on a system using 512 emulation disks causes provisioning to fail and the system launches the emergency shell in the initramfs. This happens because of an alignment bug in `sfdisk` when growing a partition. As a workaround, Ignition performs the resizing instead. (link:https://issues.redhat.com/browse/OCPBUGS-35410[*OCPBUGS-35410*])
```

Comment:
Maybe

```suggestion
* Enabling LUKS encryption on a system using 512 emulation disks causes provisioning to fail and the system launches the emergency shell in the initramfs. This happens because of an alignment bug in `sfdisk` when growing a partition. As a workaround, use Ignition to perform the resizing instead. (link:https://issues.redhat.com/browse/OCPBUGS-35410[*OCPBUGS-35410*])
```

?

Reply:
thank you 😄